Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -1018,7 +1018,7 @@ public sealed class Server : ServerBase, IDisposable {
             }
         }
 
-        private async void EnqueueItem(IDocument doc, AnalysisPriority priority = AnalysisPriority.Normal, bool enqueueForAnalysis = true) {
+        private async Task EnqueueItem(IDocument doc, AnalysisPriority priority = AnalysisPriority.Normal, bool enqueueForAnalysis = true) {
```

Comment:
👍

Reply:
Note that it's still fire-and-forget, as we very rarely want to await the entire operation (perhaps in tests...), but Mikhail confirmed that without Task here we weren't getting the first step performed synchronously, which was certainly the intent.

Reply:
It's still fire-and-forget in product, but with injection into `DoNotWait` tests can wait for those tasks to compete.